api/components/com_joomlaupdate/src/View/Updates/JsonapiView.php

```diff
@@ -98,20 +98,22 @@ public function prepareUpdate(string $targetVersion): string
      * Run the finalize update steps
      *
      * @param string $fromVersion The from version
+     * @param string $updatefileName The name of the update file
```

Comment:
```suggestion
     * @param string $updateFileName The name of the update file
```
If approved, then please change the other instances.